packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Explorer/Files/DirectoryEntry/DirectoryChildren/index.tsx

```diff
@@ -79,7 +82,11 @@ export const DirectoryChildren: React.FC<IDirectoryChildrenProps> = ({
           />
         ))}
       {sortBy(
-        modules.filter(x => x.directoryShortid === parentShortid),
+        modules
+          .filter(x => x.directoryShortid === parentShortid)
+          .filter(
+            x => !(x.directoryShortid == null && HIDDEN_FILES.includes(x.title))
```

Comment:
does this work for nested files? or it is explicitly implemented to ignore only root level files?

Reply:
Yes, only for root files 👍 

This stuff is history soon, so it is just to handle the `.eslintrc.cjs` file which is always on the root 👍